### ISSUE TITLE
iPhone 7 Plus Compatibility 

### ISSUE THREAD

**Issue by ahanank** [mullak99/extra_recipe#5]
Hey there, I just found your branch today and tried using it on i7 10.1.1. It either crashes or reboots. I have tried it like 20+ times. On the other hand, I was able to jb using the regular version in about 5 attempts. Is this branch still supposed to work as of now?

**Comment by mullak99** [comment]
Yeah it should work, I use it on my 7 Plus (iOS 10.1.1). It's a bit of a hit and miss getting it to jailbreak (just like the official one can be). Last time i rejailbreaked it worked first time, but I've had it take 20+ times too.

Same with the crashes and reboots. Sometimes it outright reboots the phone, sometimes just crashes and rarely seems to show a 'Failed' message.
